workshop/content/20-typescript/30-hello-cdk/300-cdk-watch.md

```diff
@@ -0,0 +1,177 @@
++++
+title = "CDK Watch"
+weight = 300
++++
+
+## Faster Personal Deployments
+
+{{% notice info %}} This section is not necessary to complete the workshop, but we
+recommend that you take the time to see how `cdk watch` can speed up your personal
+deployments.
+{{% /notice %}}
```

Comment:
I still like where we're putting the CDK Watch stuff, but I do think it's a bit disruptive and not necessary for people to complete the workshop, so I wanted to add this disclaimer here.

Reply:
Sure. I don't mind it 🙂.